modules/abseil-cpp/20230802.0.bcr.1/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+"""Module definition for Abseil LTS 20230802.0."""
+module(
+    name = "abseil-cpp",
+    version = "20230802.0.bcr.1",
+    compatibility_level = 1,
+)
+bazel_dep(name = "rules_cc", version = "0.0.8")
+bazel_dep(name = "platforms", version = "0.0.7")
+bazel_dep(name = "bazel_skylib", version = "1.4.1")
+bazel_dep(name = "googletest", version = "1.14.0", repo_name = "com_google_googletest")
```

Comment:
I guess this can't easily be made a dev dependency? It does have low footprint, so not much of an issue.

Reply:
Yes, I think googletest and abseil-cpp are more coupled together.